src/components/RadioButtons/RadioButtons.js

```diff
@@ -98,47 +52,72 @@ RadioButton.propTypes = {
   onFocus: PropTypes.func,
 }
 
-/**
- * RadioButtonGroup returns a stack of the above `<RadioButton>`'s.
- *
- * It is mostly form-library-agnostic, to lessen our dependency on redux-form,
- * but there are a few aspects worth mentioning.
- *
- * If this is used as the component for a Field in the shared/fields folder,
- * (it will), it can expect to receive a `value` prop (via redux-form). We can
- * then use this value to determine which radio button should receive the
- * `checked` prop. (See similar ideas on redux-form #124: https://git.io/fj9Tb)
- *
- * If we change form libraries/approaches, this component might need tweaks.
- *
- * @param {String}     name      The name of the field, e.g. 'faveColor'
- * @param {Array}      options   Props passed to the individual radios
- * @param {String}     value     The currently selected option.value
- * @param {Boolean}    disabled  Whether this radio group is disabled
- * @param {Boolean}    required  Whether this radio group is required
- * @param {...Object}  rest      Mostly redux-form Field input/meta props
- */
 export function RadioButtonGroup({
-  name,
   options,
-  value,
+  onChange,
+  formChangeHandler,
+  name = `radio-button-group-${uuidv4()}`,
+  initialValue = undefined,
+  currentValue,
+  currentError,
+  formTouched,
   disabled,
+  validator,
   required,
-  ...rest // includes redux-form props, e.g. input.onChange
+  ...rest
 }) {
-  // Passing `disabled` at the group/field level disables all options. You can
-  // also disable an individual option/radio button via `option.disabled`.
-  // Not supported yet: autoComplete, tabIndex (not obviously necessary)
+  let initialSelected
+  if (currentValue || typeof currentValue === 'boolean') {
+    initialSelected = currentValue
+  } else if (initialValue || typeof initialValue === 'boolean') {
+    initialSelected = initialValue
+  }
+
+  const [selectedValue, setSelectedValue] = useState(initialSelected)
```

Comment:
There's some known duplication between this and ButtonSelectGroup. I don't want to overdo it on this PR, so I've created a follow up task for DRY'ing that up:

https://app.asana.com/0/1137984670709817/1148024641301841/f

Reply:
🙌